packages/wp-story-editor/src/components/header/buttons/index.js

```diff
@@ -76,7 +76,7 @@ function Buttons() {
     })
   );
 
-  const isDraft = 'draft' === status;
+  const isDraft = 'draft' === status || !status;
```

Comment:
Fixes an unrelated bug I noticed during testing.

While the story is being loaded in the editor, the `status` is `undefined`.

This caused the `SwitchToDraftButton` button to briefly render before changing to the `UpdateButton`.

This fix prevents that flickering.